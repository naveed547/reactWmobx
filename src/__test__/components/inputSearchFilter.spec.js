import React from "react";
//import {mount} from "enzyme";
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import InputSearchFilter from "../../components/inputSearchFilter";
//import initialState from '../../reducers/initialState';
//import repoMock from "../data/mock";

describe("InputSearchFilter", () => {
  it('InputSearchFilter component renders correctly', () => {
      const props = {
          search: jest.fn(),
      };
      const rendered = renderer.create(
          <MemoryRouter>
              <InputSearchFilter {...props} />
          </MemoryRouter>
      );
      expect(rendered.toJSON()).toMatchSnapshot();
  });
});
  