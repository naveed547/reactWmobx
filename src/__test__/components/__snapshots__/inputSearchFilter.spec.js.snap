// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputSearchFilter InputSearchFilter component renders correctly 1`] = `
<form
  id="searchForm"
  onSubmit={[Function]}
>
  <input
    className="form-control"
    name="search"
    onChange={[Function]}
    type="search"
  />
</form>
`;
